docs/content/docs/how-to/add-mmv1-resource.md

```diff
@@ -29,6 +29,60 @@ contains the fields of the resource based on how it behaves in the API, and
 behaviour. Not all fields will need to be added to `terraform.yaml`- only add an
 entry for your field if you need to configure one of the available option(s).
 
+## Adding a product
+
+Product definitions are USUALLY a particular API domain (like `compute.googleapis.com`) and can contain
+on or more objects (Terraform resources). The product definitions (`!ruby/object:Api::Product`) are located 
+in `mmv1/products/<product>/api.yaml`. 
+
+|       Name        |  Type  |                                   Description                                   |                                   Notes                                    | Required |
```

Comment:
I'm a little concerned about adding this table here manually, because it will be so easy to get out of sync. I would suggest just linking to the ruby source files, or perhaps there is a simple way to autogenerate the table.